src/e3/diff.py

```diff
@@ -92,7 +94,9 @@ def diff(
                 line for line in contents[k] if re.search(ignore, line) is None
             ]
 
-    return "".join(unified_diff(contents[0], contents[1], item1name, item2name, n=1))
+    return "".join(
+        unified_diff(contents[0], contents[1], item1name, item2name, n=context)
```

Comment:
Just a little remark. If we use named variable for `n`, we could also name the other name with default values (*item1name* and *item2name* are `fromfile` and `tofile` if I am not wrong).
Still this is just a remark.

Reply:
The reason `n` is named is that it can't be positional, as the code skips over `fromfiledate` and `tofiledate` - but the other arguments can be positional. I guess we should refer to the AdaCore Python Style Guide (is there such a thing? I found one for Ada but not Python...) to know what we should do.

Reply:
Nah, as said, it is just a remark. I also tend to use names everywhere :) I just wanted to have @glacambre advice on that :)

Reply:
Done.

I was about to suggest enforce this style rule with automation, but it seems that despite the plethora of python linters/static analyzers, there's no pre-existing rule that checks for this. It seems implementabale with CodeQL though.